rational/tests/convert.rs

```diff
@@ -54,11 +52,11 @@ fn test_from_f32() {
     assert_eq!(RBig::try_from(2.25f32), Ok(rbig!(9 / 4)));
     assert_eq!(RBig::try_from(-2.25e4f32), Ok(rbig!(-22500)));
     assert_eq!(RBig::try_from(1.1773109e-2f32), Ok(rbig!(12345 / 1048576)));
+    assert!(RBig::try_from(core::f32::consts::PI).is_ok());
 
     assert_eq!(RBig::try_from(f32::INFINITY), Err(OutOfBounds));
     assert_eq!(RBig::try_from(f32::NEG_INFINITY), Err(OutOfBounds));
     assert_eq!(RBig::try_from(f32::NAN), Err(OutOfBounds));
-    dbg!(RBig::try_from(3.1415926f32).unwrap());
```

Comment:
I also did this while I was at it, but I'm not sure if you had that in mind for that case